index.html

```diff
@@ -163,10 +220,50 @@ <h4>LET'S CONNECT</h4>
           <li>Frameworks</li>
           <li>Skills</li>
         </ul>
-        
       </div>
     </section>
-  
-  
-</body>
-</html>
\ No newline at end of file
+
+    <section class="contact-me">
+      <h2 class="title-contact">Contac me</h2>
+      <p class="description-contact">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/mlezgrdz" method="post">
+        <ul class="input-list">
+          <li class="form-input">
+            <input
+              type="text"
+              name="name"
+              placeholder="Enter your name"
+              required
+              maxlength="30"
+              class="text-input"
+            />
+          </li>
+          <li class="form-input">
+            <input
+              type="email"
+              name="_replyto"
+              placeholder="Enter your email"
+              required
+              class="text-input"
+            />
+          </li>
+          <li class="form-input">
+            <textarea
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements: Please ensure that the message field accepts up to 500 chars. You can do this by adding the `maxlength="500"` attribute

Reply:
Hi, @lily-coder I resubmitted the code including maxlength in textarea now. But in my dashboard instead of a second review now appears a green button like when the activity was finished successfully. Let me know if now I can merge the pull request.